addons/hide-rows/userscript.js

```diff
@@ -0,0 +1,52 @@
+export default async function ({ addon, global, console }) {
+  // Updates the page if any settings are changed
+  addon.settings.addEventListener("change", Change());
+
+  // Main Function
+  function Change(argument) {
+    // What's Happening
+    if (addon.settings.get('whs_ha') === true) {

```

Comment:
```suggestion
    if (addon.settings.get('whs_ha')) {
```
You don't need to check if `true == true`.